partner_bank_report_rel/models/ir_actions_report.py

```diff
@@ -0,0 +1,16 @@
+from odoo import models, fields
+
+
+class IrActionsReport(models.Model):
+    """Extend to field partner_bank_ref_ids."""
+
+    _inherit = 'ir.actions.report'
+
+    partner_bank_ref_ids = fields.Many2many(
```

Comment:
We will have to keep in mind that those bank account's might be from other company as well (in multi-company environment). And doesn't matter partners are shared or not, bank accounts should be filtered by current company's partner.

Reply:
Mhm, will check that
But it should not matter. If it has access to it, it will see it, if not it will not. Because bank accounts are filtered by companies whether partner is shared or not by default.

Though one more thing is missing here, to make it better. We need extra module `base_is_company_partner`, which would show us if partner is partner that is set on res.company. Thing is Odoo has needed relation via field `ref_company_ids`, but for some reason its added on `account` module, so you can only use it if account module is installed. But we need such relation not just for accounting.

So will need to create some independent logic (which depends on base only). Then we could use it to know if bank account is ours or some customer. And then could also identify partner in credentials template, knowing is it us or customer.